plugins/receiver/grpc/nativetracing/receiver.go

```diff
@@ -52,6 +53,9 @@ func (r *Receiver) RegisterHandler(server interface{}) {
 	v3.RegisterTraceSegmentReportServiceServer(r.Server, r.service)
 }
 
+func (r *Receiver) RegisterSyncProcessor(_ api.SyncProcessor) {
```

Comment:
This is API definition, right? It seems missing comments.

Reply:
It has been add comments on the `Receiver` interface. https://github.com/apache/skywalking-satellite/pull/53/files#diff-d7d2624abe5a66c8d5e294b65af7c537db3cece721746cec346e4582614d128eR37-R38